integrations/lib/multiwoven/integrations/core/constants.rb

```diff
@@ -26,6 +26,8 @@ module Constants
         }
       }.freeze
 
+      ZENDESK_TICKETING_URL = "https://multiwoven-test.zendesk.com/api/v2/"
```

Comment:
is ZENDESK_TICKETING_URL constant across all customers?

Reply:
No. It depends on each customer's subdomain. I created the multiwoven-test subdomain for my tests but I will make this dynamic so it reads something like "https://#{subdomain}.zendesk.com/api/v2".

Thanks for calling this out.

Reply:
Also please add this configuration in `spec.json` as this will change for every user

Reply:
Done @karthik-sivadas.